src/gmres_e.m

```diff
@@ -0,0 +1,426 @@
+function [x, flag, relresvec, time] = ...
+    gmres_e(A, b, m, k, tol, maxit, xInitial, varargin)
+    % GMRES-E algorithm
+    %
+    %   GMRES-E is a modified implementation of the restarted
+    %   Generalized Minimal Residual Error or GMRES(m) [1], performed by
+    %   appending 'k' eigenvectors corresponding to a few of the smallest
+    %   eigenvalues in magnitude for each outer iteration. In practice,
+    %   the approximate eigenvectors are the harmonic Ritz vectors
+    %   associated to the harmonic Ritz values per outer iteration.
+    %
+    %   Signature:
+    %   ----------
+    %
+    %   [x, flag, relresvec, time] = ...
```

Comment:
Maybe the signature can fit in one line?

Reply:
Agree!